doc/object.md

```diff
@@ -288,5 +288,47 @@ Napi::Value Napi::Object::operator[] (uint32_t index) const;
 
 Returns an indexed property or array element as a [`Napi::Value`](value.md).
 
+## Iterator
+
+Iterators expose an `std::pair<...>`, where the `first` property is a
+[`Napi::Value`](value.md) that holds the currently iterated key and the
+`second` property is a [`Napi::Object::PropertyLValue`](propertylvalue.md) that
+holds the currently iterated value. Iterators are only available if C++
+exceptions are enabled (by defining `NAPI_CPP_EXCEPTIONS` during the build).
+
+### Constant Iterator
+
+In constant iterators, the iterated values are immutable.
+
```

Comment:
Here you should document the interface exposed by `Napi::Object::const_iterator`.